src/langgraph_google_alloydb_pg/engine.py

```diff
@@ -454,6 +477,8 @@ async def _ainit_checkpoint_table(
         
         async with self._pool.connect() as conn:
             await conn.execute(text(create_checkpoints_table))
+            await conn.commit()
+        async with self._pool.connect() as conn:
```

Comment:
Were these 2 lines needed to create both tables? I would expect we could call both executes then one commit like in the previous commit. If this doesn't work we should be able to execute and commit both tables within the same connection so line 481 isn't needed.